crates/aptos-genesis/src/config.rs

```diff
@@ -35,17 +31,33 @@ pub struct Layout {
     pub users: Vec<String>,
     /// ChainId for the target network
     pub chain_id: ChainId,
+    /// Whether to allow new validators to join the set after genesis
+    #[serde(default)]
+    pub allow_new_validators: bool,
+    /// Minimum stake to be in the validator set
+    pub min_stake: u64,
+    /// Minimum number of seconds to lockup staked coins
+    pub min_lockup_duration_secs: u64,
+    /// Duration of an epoch
+    pub epoch_duration_secs: u64,
+    /// Initial timestamp for genesis validators to be locked up
+    pub initial_lockup_timestamp: u64,
+    /// Min price per gas unit
+    pub min_price_per_gas_unit: u64,
```

Comment:
We should align on those variable naming in https://github.com/aptos-labs/aptos-core/pull/1413/files to make them consistent